src/components/Calculator.js

```diff
@@ -1,176 +1,128 @@
-        >
-          AC
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-light btn-toogle-sign"
-        >
-          +/-
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-light btn-percentage"
-        >
-          %
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-accent btn-divider"
-        >
-          /
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-accent btn-multiply"
-        >
-          *
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-accent btn-substract"
-        >
-          -
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClick}
-          className="btn-accent btn-plus"
-        >
-          +
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-one"
-        >
-          1
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-two"
-        >
-          2
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-three"
-        >
-          3
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-four"
-        >
-          4
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-five"
-        >
-          5
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-six"
-        >
-          6
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-seven"
-        >
-          7
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-eight"
-        >
-          8
-        </button>
-        <button
-          type="button"
-          onClick={this.btnOnClickOperator}
-          className="btn-light btn-nine"
-        >
-          9
-        </button>
+        <div className="result">
+          {total || ''}
+          {operation || ''}
+          {next || ''}
+        </div>
+        <Button
+          typeBtn="0"
+          btnOnClick={this.btnOnClick}
+          btnClass="btn-light btn-zero"
+        />
+        <Button
+          typeBtn="."
+          btnOnclick={this.btnOnClick}
+          btnClass="btn-light btn-dot"
+        />
```

Comment:
- [ ] currently this button is not clicking, kindly fix this

Reply:
This change has been done well